src/mame/audio/mw8080bw.h

```diff
@@ -52,13 +53,20 @@ class gunfight_audio_device : public device_t
 	gunfight_audio_device(machine_config const &mconfig, char const *tag, device_t *owner, u32 clock = 0);
 
 	void write(u8 data);
+	DECLARE_WRITE_LINE_MEMBER(left_shot_w);
+	DECLARE_WRITE_LINE_MEMBER(right_shot_w);
+	DECLARE_WRITE_LINE_MEMBER(left_hit_w);
+	DECLARE_WRITE_LINE_MEMBER(right_hit_w);
```

Comment:
These don’t seem to serve a purpose, and even if they did they should be private.

Reply:
My remark above applies here as well.